packages/gatsby-remark-prismjs/README.md

```diff
@@ -139,9 +139,10 @@ This is some beautiful code:
     ]
     ```
 
-You can also add line highlighting. It adds a span around lines of code with a
-special class `.gatsby-highlight-code-line` that you can target with styles. See
-this README for more info.
```

Comment:
Note that I removed this reference to a README because there was no link, I checked the history and it was never linked.